src/views/Recorder/BrowserEventLog.tsx

```diff
@@ -16,31 +43,91 @@ function EventIcon({ event }: EventIconProps) {
     case 'page-reload':
       return <UpdateIcon />
 
+    case 'click':
+      return <TargetIcon />
+
     default:
       return exhaustive(event)
   }
 }
 
+interface PageNavigationDescriptionProps {
+  event: PageNavigationEvent
+}
+
+function PageNavigationDescription({ event }: PageNavigationDescriptionProps) {
+  const url = (
+    <Tooltip content={event.url}>
+      <strong>{event.url}</strong>
+    </Tooltip>
+  )
+
+  switch (event.source) {
+    case 'interaction':
+      return <>Navigated to {url} by interacting with the page.</>
+
+    case 'script':
+      return <>Navigated to {url} by a script.</>
+
+    case 'address-bar':
+      return <>Navigated to {url} using the address bar.</>
+
+    case 'history':
+      return <>Navigated to {url} using the browser history.</>
+
+    default:
+      return exhaustive(event.source)
+  }
+}
+
+function getButtonDescription(event: ClickEvent) {
+  switch (event.button) {
+    case 'left':
+      return 'Clicked'
+
+    case 'middle':
+      return 'Middle-clicked'
+
+    case 'right':
+      return 'Right-clicked'
+
+    default:
+      return exhaustive(event.button)
+  }
+}
+
+interface ClickDescriptionProps {
+  event: ClickEvent
+}
+
+function ClickDescription({ event }: ClickDescriptionProps) {
+  const modifiers = getModifierKeys(event.modifiers)
+  const button = getButtonDescription(event)
+
+  const clickedText = modifiers.concat(button).join(' + ')
```

Comment:
Might be a good place for the [Kbd](https://www.radix-ui.com/themes/docs/components/kbd) component (font size consistency should probably be improved tho):

<img width="704" alt="image" src="https://github.com/user-attachments/assets/05bd7d7d-eb21-45e6-9df4-71185c1b0c94" />

Reply:
Awesome! I did not know that was a thing. Thank you! 🙏